Account/FxADeviceRegistration.swift

```diff
@@ -54,7 +54,7 @@ open class FxADeviceRegistration: NSObject, NSCoding {
 
     public convenience required init(coder: NSCoder) {
         let id = coder.decodeObject(forKey: "id") as! String
-        let version = coder.decodeObject(forKey: "version") as! Int
+        let version: Int = coder.decodeObject(forKey: "version") as! Int? ?? 0
```

Comment:
Yea this was a startup crash for me. I thought you fixed it by using `decodeInt(forKey:)`